doc/source/UsingSpackEnvironments.rst

```diff
@@ -5,6 +5,14 @@ Using spack-stack environments
 
 The following tutorial assumes you have a functioning spack-stack environment installed local to your system. This environment is provided on platforms described in :numref:`Section %s <Preconfigured_Sites>`. If you intend to run spack-stack on your developer machine or on a new platform, you can create an environment using the steps described in :numref:`Section %s <NewSiteConfigs>`.
 
+There are three steps in setting up a usable development environment.
+The first is to load the spack-stack environment and the second is to create a python virtual environment that is based on the python executable included within the spack-stack installation.
```

Comment:
Might be useful to mention here that as many as possible of the spack-stack Python modules (not just the executable) should be loaded when using/installing additional packages in the virtual environment.

Reply:
Also, when you push updates, please include `[skip ci]` in the commit message - no need to run the tests for doc updates only.

Reply:
Added a note encouraging the use of the spack-stack installed python modules.